packages/lib-feature-flags/src/FeatureProvider.js

```diff
@@ -0,0 +1,107 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
Do we need use this rule in current module?

Reply:
@andrei-oksentyk Seems like you're right, thanks! :)